### DISCUSSION TITLE
detach tab works ok but re-attach/dock doesn't?

### DISCUSSION THREAD

**Discussion by gthbusr0** [dbeaver/dbeaver#13547] [Q&A] [answered]
how to dock or re-attach a sql editor tab that has been detached? so far, having to close & re-open the tab.

**Comment by p91paul** [comment, marked as answer]
I assume you mean you dragged the tab out in its own window. If so, you just need to drag it back in the main window!

**Comment by gthbusr0** [comment]
giving me grief while doing that as it always attempts to split either vertically or horizontally..wondering what might I be missing ...

**Reply by p91paul** [reply]
Point your mouse to the tab bar in the destination window while dropping. It must be exactly next to another tab button, anywhere else triggers splitting.

**Reply by gthbusr0** [reply]
that did it, Paul! thx
